rmf_fleet_adapter/include/rmf_fleet_adapter/agv/EasyFullControl.hpp

```diff
@@ -316,6 +317,15 @@ class EasyFullControl::RobotConfiguration
   /// Set the minimum lane length.
   void set_min_lane_length(std::optional<double> distance);
 
+  /// Get the idle behavior.
+  ///
+  /// If std::nullopt is used, then the fleet-wide default finishing request
+  /// will be used.
+  std::optional<rmf_task::ConstRequestFactoryPtr> finishing_request() const;
```

Comment:
I recall you mentioned your intention to shift completely to using `idle_behavior` in the future, so I've used the term in most new comments in this PR, and chose to stick to `finishing_request` in the API signatures to stay consistent with the current library. It seems weird to call it `finishing_request` in FleetConfiguration but `idle_behavior` in RobotConfiguration. On hindsight this discrepancy in comments and API signature is not very helpful and could be confusing.

Looking at how much `finishing_request` is already being used throughout the fleet adapter library, its python bindings, and in all fleet config yamls, I personally feel that it might be more meaningful to make clean switch to `idle_behavior` in next gen RMF.